src/components/common/Map.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ReactMapboxGl from 'react-mapbox-gl';
+
+const Map = () => {
+  const mapBoxToken = 'pk.eyJ1Ijoic2pheW1vb24xNSIsImEiOiJjajE2ejhjMXgwM3YxMndtejdpbnlkbWp0In0.HAeIBMg25HSE8zHyNHMccg';
```

Comment:
Let's store this token in a different file that is used specifically to load settings data and just import it.